wallet-connect-test-bench/front-end/src/constants.ts

```diff
@@ -77,6 +77,12 @@ export const VIEW_RETURN_VALUE_SCHEMA =
 
+export const BASE_64_SCHEMA_WITH_INIT_U8_PARAMETER =
```

Comment:
It's a bit unclear where this is coming from. There seem to be two different contracts now.

At the very least we should document how to construct this.

A better option would be to not have two contracts. Instead we could just have one contract that takes an optional parameter. You can check if there was any parameter specified in the init function using https://docs.rs/concordium-std/latest/concordium_std/trait.HasSize.html and then act accordingly. 

Does this make sense?

Reply:
I adjusted the `test_bench_smart_contract` init function. Which now expects an optional `u16` input parameter.